### PR TITLE
Update system_install_ubuntu_1604.rst

### PR DESCRIPTION
**Pull request by tliesner**
Hi,

as Tobias requested, I corrected some minor glitches in this document.
Please confirm and pull.

Kind regards,
Tom

### CODE REVIEW

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 1]
```diff
@@ -143,7 +143,7 @@ PHP 5.6 (supported)
 
     $ sudo apt-get update
     $ sudo apt-get install mongodb
-    $ sudo apt-get install php5-mongo
+    $ sudo apt-get install php-mongo
```
Comment (mickaelandrieu) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: In this case, you are in the "php 5.6" section.

How about add this instructions in the "PHP 7.0 (experimental)" section?

````
sudo apt-get remove php5-mongo
sudo apt-get install php-mongo
````

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 2]
```diff
@@ -159,11 +159,13 @@ You'll have to install the **new** Mongo PHP extension_ and enable it:
     $ sudo phpenmod mongodb
 
 .. _adapter: https://github.com/alcaeus/mongo-php-adapter
+.. _composer: https://getcomposer.org
 
-Finally, you have to install the Mongo PHP adapter_:
+Finally, you have to install the Mongo PHP adapter_ using the dependancy management tool composer_:
```
Comment (mickaelandrieu) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]:
```diff
- dependancy management tool
+dependency manager
```

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 1 above)
Comment (tliesner) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Hmm. I did not try to to go the PHP 7.0 way, but there is plainly no package named php5-mongo in ubuntu 16.04.. Hence, it cannot be removed ;)
The following should be enough:
`sudo apt-get install php-mongo`

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 2 above)
Comment (tliesner) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: perfectly fine.

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 1 above)
Comment (mickaelandrieu) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: oh ? sorry :) the right package name is ``php5.6-mongo``

Comment (tliesner) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Are you sure? I can't find a package of that name in http://packages.ubuntu.com/de/xenial/allpackages
All there is a package named `php-mongodb` and that has a PHP 7.0 dependency:
[http://packages.ubuntu.com/de/xenial/php-mongodb](url)

Comment (mickaelandrieu) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Well, I've installed a fresh ubuntu 16.04 and cant found any ``php5.6-mongo`` package, but ``php5-mongo`` exists .. in 14.04.

Sorry for the disturbing, we learn every day :)

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 3]
```diff
@@ -159,11 +159,13 @@ You'll have to install the **new** Mongo PHP extension_ and enable it:
     $ sudo phpenmod mongodb
 
 .. _adapter: https://github.com/alcaeus/mongo-php-adapter
+.. _composer: https://getcomposer.org
 
-Finally, you have to install the Mongo PHP adapter_:
+Finally, you have to install the Mongo PHP adapter_ using the dependancy manager composer_:
```
Comment (mickaelandrieu) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Sorry, I wasn't clear on my review:

```diff
- dependancy
+ dependency
```

Then it's perfect to me :)

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 4]
```diff
@@ -2,7 +2,7 @@ System install on Ubuntu 16.04
 ==============================
 
 Here is a quick guide to setup the :doc:`system_requirements` on Ubuntu 16.04. This guide will help you to install all
-the packages and modules needed for Akeneo PIM, then configure it to match your local installation.
+the packages and modules needed for Akeneo PIM on a newly installed ubuntu 16.04 system, then configure it to match your local installation.
```
Comment (jmleroux) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Why _newly installed_ ?

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst [hunk 5]
```diff
@@ -159,11 +159,13 @@ You'll have to install the **new** Mongo PHP extension_ and enable it:
...
 
 .. _adapter: https://github.com/alcaeus/mongo-php-adapter
+.. _composer: https://getcomposer.org
 
-Finally, you have to install the Mongo PHP adapter_:
+Finally, you have to install the Mongo PHP adapter_ using the dependency manager composer_:
 
 .. code-block:: bash
 
+    $ sudo apt-get install composer
```
Comment (jmleroux) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: I'm not sure this is the recommend way to use composer. I'm not a big fan of installing composer via apt-get.

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 4 above)
Comment (MarieMinasyan) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: _freshly installed_ would be nicer :)

Comment (MarieMinasyan) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: _and then configure the application to match_

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 5 above)
Comment (damien-carcel) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Indeed, it is always a very old version of composer. We should install it as it is recommended in composer documentation.

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 4 above)
Comment (damien-carcel) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: I think it means no apache/php

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 5 above)
Comment (MarieMinasyan) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: I agree with you guys, this can be rather confusing

Comment (tliesner) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: What is the recommended way to install composer? It worked that way.

Comment (damien-carcel) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: https://getcomposer.org/download/
The problem is not that it does not work, but that your composer is old, that's all.

Comment (MarieMinasyan) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 4 above)
Comment (tliesner) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: Would you see this guide helpful in all situations which could occur when installing akeneo on a system with unknown usage history?

**Comment by mickaelandrieu** [comment]
By the way, thank you @tliesner for your contribution we appreciate a lot your efforts!

File: developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst (hunk 5 above)
Comment (tliesner) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: By following the argument that composer will be too old, you should also consider changing the way to install mongodb. When selecting the PHP7.0-way one should install it via pecl and when choosing to go the PH5.6-way one should install it via apt-get although pecl would work as well...

Comment (damien-carcel) [inline developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst]: That is not the same problem. Composer is independent and can/should be the most recent possible.
php-mongo is a PHP extension and must be installed along other PHP extensions. The most secure way to do that is to use packages that are made to run together. Now, if you want more recent packages for PHP and its extensions, you can use an external repository, like https://deb.sury.org/ (it is made by the official maintainer of PHP for Debian, so trustable).

**Comment by mickaelandrieu** [comment]
@tliesner thanks for your help, closed in favor of https://github.com/akeneo/pim-docs/pull/587

**Pull request closed without merging**